en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md

```diff
@@ -115,7 +115,7 @@ before `</Handlers>`.
     you need to maintain a customized velocity template file that needs to be manually merged when you upgrade your 
     product to a newer version. Therefore, it is recommended to use custom Handlers when you wish to specify the exact 
     order of execution of JARs as this can not be done with 
-    [Mediators](../../../extensions/adding-mediation-extensions) .
+    [Mediators](../../../extensions/adding-mediation-extensions).
```

Comment:
Shall we use {{base_path}} here as well?

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/5293/commits/67ab08363c38e18c4129b53f63ee12266aedc5b3